### PR TITLE
XFAIL two failing projects

### PR DESCRIPTION
**Pull request by benlangmuir**

### CODE REVIEW

**Comment by benlangmuir** [comment]
@swift-ci please test with toolchain

**Review by shahmishal — APPROVED** [review]

**Comment by shahmishal** [comment]
```
00:02:45.048 + ./swift-source-compat-suite/utils/format_projects_list.py swift-source-compat-suite/projects.json
00:02:45.244 Traceback (most recent call last):
00:02:45.244   File "./swift-source-compat-suite/utils/format_projects_list.py", line 61, in <module>
00:02:45.244     sys.exit(main())
00:02:45.244   File "./swift-source-compat-suite/utils/format_projects_list.py", line 47, in main
00:02:45.244     ).decode(open(args.project_index).read()),
00:02:45.244   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 364, in decode
00:02:45.245     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
00:02:45.245   File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 380, in raw_decode
00:02:45.245     obj, end = self.scan_once(s, idx)
00:02:45.245 ValueError: Expecting property name: line 3321 column 11 (char 85805)
```

File: projects.json [hunk 1]
```diff
@@ -3307,6 +3314,12 @@
       {
         "action": "BuildSwiftPackage",
         "configuration": "release",
+        "xfail": [
+          {
+            "issue": "rdar://83452858",
+            "compatibility": ["5.0"],
```
Comment (xedin) [inline projects.json:3320]: I think this is the issue here, json doesn't allow `,` without subsequent elements.

Comment (benlangmuir) [reply projects.json:3320]: 😭 I'm used to having a more lenient json parser.  Thanks

**Comment by benlangmuir** [comment]
@swift-ci please test with toolchain

**Pull request merged**